packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -153,6 +154,8 @@ env.tracker.register(
   })
 );
 
+env.tracker.register(createWithdrawalAssertions(env.nodes[0].id));
```

Comment:
I'm not sure how this works. I suppose we find a way to include withdrawals in the state, then later on assert that our block includes that withdrawals, not sure if we can do that through as it depends on `nextWithdrawalIndex` in the state through

Reply:
nvm, found `withEth1Credentials: true` below

Reply:
Yes it took me a while also to figure out how to make withdrawals work.